### ISSUE TITLE
Document

### ISSUE THREAD

**Issue by ghost** [kryshtalandrew/kryshtalandrew.github.io#1]
НЕЗАЧЕТ, визуально не совсем совпадает.

Хочу видеть серый фон у страницы, центральный блок с белым фоном по центру, скорее всего надо ему ширину задать. ну и тень)) Исправить!!!

хотеллось бы видеть нормальную структуру папок, css - для файлов стилей, img - для картинок. Исправить!!!

картинку выровнять через text-align  Исправить!!!

я не хочу видеть ни один елемент hr, убрать его и заменить на бордер. Критично! Исправить.

я не хочу видеть ни один елемент br, убрать его и заменить на margin с использованием классов. Критично! Исправить.

Проверено - 21.10.2015.

**Comment by kryshtalandrew** [comment]
исправил все ошибки

**Comment by ghost** [comment]
Незачет! Не воровать чужие домашки!\ Если  воруем, то заметать следы! Зачем в низу станицы блок с кдассом "linedown"??

Зачем поставил бордер радиус для блока с классом photo?

Ширину картинки задать через цсс.
Проверено 24.10.2015

**Comment by kryshtalandrew** [comment]
да, спалился :D зато, стал еще опытнее. бордер забыл удалить ахах :DDDDD

исправлено.

**Comment by ghost** [comment]
Зачет!
